### ISSUE TITLE
tgt.io.write_to_file does not escape double quotes

### ISSUE THREAD

**Issue by lschilli** [hbuschme/TextGridTools#6]
The following code creates a TextGrid file that cannot be read by praat:

```python
import tgt

tg = tgt.TextGrid()
tier = tgt.IntervalTier(name="tiername")
iv = tgt.Interval(0, 1, '"')
tier.add_annotation(iv)
tg.add_tier(tier)
tgt.io.write_to_file(textgrid=tg, filename="invalid.TextGrid", format="long")
```

**Comment by hbuschme** [comment]
Thanks for the bugreport. The issue has been fixed in release 1.3.2 (commit `0d636b`). An update version is now also available at the Python package index (https://pypi.python.org/pypi/tgt/1.3.2) and can be installed via `pip install tgt --upgrade`.

**Comment by lschilli** [comment]
Great, thanks!
